Gruntfile.js

```diff
@@ -28,7 +28,7 @@ module.exports = function(grunt) {
       coverage: {
         src: "src/test/**/*.js",
         options: {
-          istanbulOptions: [ "--include-all-sources" ]
+          includes: [ "src/**/*.js" ]
```

Comment:
As far as I can tell (and I played around with this), if you explicitly specify the files to include, it will judge coverage based on all those files. If I just left it with --include-all-sources, it was including things like the Gruntfile and also what was in the sample app (which I'm not quite ready to add tests yet for). I'll let you know if I discover if I was mistaken, because I certainly want to know total coverage, not just coverage for the files covered.